Update support-for-windows-10.md

JohnRajunas opened the pull request:
I am proposing three specific sets of changes to this topic. The first proposed change is to stop using the term "version" in reference to Windows 10, replacing it with the term "release" (with release used as a noun and not a verb).  

The purpose of this change is to insure we do not imply that Windows 10 has "multiple versions" when using the word “version” with Windows 10, but instead insure we are enforcing the idea that Windows 10 is in fact the last "version" of Windows (there is no "Windows 11" coming), however new capabilities and features are added to Windows over time through Feature Update "releases", where a release "becomes available" or "is generally available" (“available” is used to avoid writing that a “release” (noun) is being "released" (verb)).

The second proposed change is to update the chart and references under the “Windows as a Configuration Manager client” section to reference the support dates for the SCCM Current Branch (CB) version releases, and to update the icons for SCCM “1606” now that it has reached it’s “End of Support milestone”.  

These changes are realized through proposed changes to the icons for SCCM “1606” in the chart itself, as well as references to the actual end of Support milestone as text under each release described in the chart.  Additionally, a bullet referring readers to the Document Library topic titled “Support for System Center Configuration Manager current branch versions” is also added.  Since “support” of Windows 10 as a SCCM “client” depends on the status of each SCCM CB version release, I believe convenient reference from this topic to the “end of support” date topic is important.

The third proposed change is to remove the references to "CB" and "CBB" in relation to Windows 10 "releases". 

Documentation and discussions with Commercial Customers on the topic of Windows as a Service are moving away from the use of the "Current Branch (CB)" and "Current Branch for Business" (CBB) terminology.  Instead we (Microsoft Commercial Technical Field) are tasked with working to focus our customers on understanding that "CBB" is simply a development milestone in the life of a Windows 10 release, and they should be deploying each Semi-Annual Channel Feature Update release for Windows 10 using a process and timing that makes sense for their organization, rather than automatically having the organizational process being tied to waiting for the “CBB” development milestone just because it is titled “Current Branch for Business”. 

It is important we de-emphasize CBB as a deployment milestone for commercial customers, noting that CBB is actually a milestone in the code development of an individual Windows release that is based on a certain Quality Update being available and deployed.  We need customers to understand there is really no significant difference to Windows 10 code except for the installation of that Quality Update release that designates the CBB milestone (for example, Win 10 "1607" reaches "CBB" when the Quality Update detailed by KB 32000970 is deployed ). 

However, since SCCM CB release support for Windows 10 can be different before or after the Windows 10 release code reaches the CBB development milestone, I am proposing that any references made to CBB are avoided, and instead a reference made to the Quality Update for the Windows 10 release that brings the code to “CBB”.  This could appear in future versions of this document as a note, such as "NOTE: Backwards Compatibility Support for Windows 10 “1607” ends once Windows 10 "1607" is serviced using the Quality Update released under KB 32000970, made generally available on 2016-11-08)

I hope these proposed changes will be considered.

msftclas commented:
@JohnRajunas,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

arob98 commented:
@Brenduns For your review

Brenduns commented:
@JohnRajunas,
Thanks for the feedback John. This topic is under active review by our engineering team, and as such we are on hold to accept these changes.

arob98 commented:
@JohnRajunas We incorporated the changes you recommended via other PRs while working with engineering. I'm closing this PR, but know that we incorporated your feedback. Appreciate the contribution as it sparked good discussion that led to updates.

JohnRajunas commented:
Thanks Angela - I did see the document got updated! – I am glad my comments were helpful.  Thanks for the opportunity to contribute.


John Rajunas  |  Modern Desktop Technology Specialist
john.rajunas@microsoft.com<mailto:john.rajunas@microsoft.com> (e-mail & Skype)    |   (585) 203-3027 (Phone)   |   Syracuse, NY USA   |  Windows IT Pro<https://docs.microsoft.com/en-us/windows/#pivot=it-pro>
-  -  -  -  -  -  -  -  -  -  -
The information in this message is intended solely for the designated addressees and is provided “AS IS”, without warranties, and confers no rights.
This message was sent using my Microsoft 365 powered device, and is best viewed in an HTML format

[cid:image001.png@01CFFF68.F01F79B0]



From: Angela Robertson [mailto:notifications@github.com]
Sent: Tuesday, August 22, 2017 6:53 PM
To: MicrosoftDocs/SCCMdocs <SCCMdocs@noreply.github.com>
Cc: John Rajunas <John.Rajunas@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [MicrosoftDocs/SCCMdocs] Update support-for-windows-10.md (#256)


@JohnRajunas<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjohnrajunas&data=02%7C01%7Cjohn.rajunas%40microsoft.com%7C97652d2eb0904349725308d4e9b09884%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636390392061942443&sdata=4kUqTQFls8yN6g%2FRX3t6jh4tt0875lKToz7OUMhKZ14%3D&reserved=0> We incorporated the changes you recommended via other PRs while working with engineering. I'm closing this PR, but know that we incorporated your feedback. Appreciate the contribution as it sparked good discussion that led to updates.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FSCCMdocs%2Fpull%2F256%23issuecomment-324173622&data=02%7C01%7Cjohn.rajunas%40microsoft.com%7C97652d2eb0904349725308d4e9b09884%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636390392061942443&sdata=ZxN%2BOkPwE5zMZ59at06tXsH28hXdYrBkHusNXBH1mPs%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAc12WNCn9ZUQ0Ryl8F1GFkbH1r777PbCks5sa1vkgaJpZM4Oi4zQ&data=02%7C01%7Cjohn.rajunas%40microsoft.com%7C97652d2eb0904349725308d4e9b09884%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636390392061942443&sdata=PdNrWbcxwOtQ9R1oqRVPMCAKO1YKwX8n2Vo17VdraYs%3D&reserved=0>.
